src/docfx/config/Config.cs

```diff
@@ -320,6 +320,8 @@ internal class Config : PreloadConfig
 
         public JoinTOCConfig[] JoinTOC { get; private set; } = Array.Empty<JoinTOCConfig>();
 
+        public string[] SplitTOC { get; private set; } = Array.Empty<string>();
```

Comment:
This could be defined as `HashSet<PathString>` to make `SplitToc` become a simple `Contains` check.
```suggestion
        public HashSet<PathString> SplitTOC { get; private set; } = new HashSet<PathString>();
```

Reply:
+1, should this be case insensitive?

Reply:
I thought it should be case sensitive for two file named api/toc.yml and API/TOC.YML can be placed into the build scope, they are different files.

Reply:
@AngryBerryMS , the value here is a file path. File path is currently case insensitive on windows and mac but case sensitive on linux (the behavior might change in the future). This logic is encapsulated in `PathString` and it is a good practice to use the strongly typed `PathString` to represent file path wherever you can.